apis/r/R/Factory.R

```diff
@@ -6,10 +6,11 @@
 #' @param index_column_names Index column names passed on to DataFrame$create()
 #' @param platform_config Optional platform configuration
 #' @param tiledbsoma_ctx Optional SOMATileDBContext
+#' @param tiledb_timestamp Optional POSIXct for TileDB timestamp
```

Comment:
Maybe write as 'Optional Datetime (i.e. POSIXct) variable with TileDB timestamp.  Not everybody will be familiar with term POSIXct.